src/EditDialog.js

```diff
@@ -160,58 +181,37 @@ export default function EditDialog({ open, id, onClose }) {
         <DialogContent className={classes.content}>
           <TextField
             label={formatMessage(msgs.title_label)}
-            value={
-              value === undefined
-                ? ""
-                : title === undefined
-                ? value.title
-                : title
-            }
+            value={title}
+            InputLabelProps={{ shrink: !!title }}
```

Comment:
necessary to prevent the floating label from overlapping with a non-empty input value (see [docs](https://v4.mui.com/components/text-fields/#shrink))